monitoring/healthz/src/Router.tsx

```diff
@@ -37,7 +35,7 @@ const routeList: RouteObject[] = [
       {
         path: 'trending_playlists',
         element: (
-          <DiscoveryTrending trendingEndpoint="/v1/playlists/trending/BDNxn" />
```

Comment:
dumb question but does the client need this change too? why would healthz need this route specifically?

Reply:
no dumb question! not sure i understand it though.

audius-client (audius.co) uses an optimizely env var to control this so no code change is needed. healthz just added this because it's "internal" facing and easier to cycle on